src/Views/Form/Templates/Sequoia/views/social-sharing.php

```diff
@@ -0,0 +1,46 @@
+<?php if ( isset( $options['thank-you']['sharing'] ) && $options['thank-you']['sharing'] === 'enabled' ) : ?>
+	<div class="social-sharing">
+		<p class="instruction">
+			<?php echo $options['thank-you']['sharing_instruction']; ?>
+		</p>
+		<div class="btn-row">
+			<!-- Use inline onclick listener to avoid popup blockers -->
+			<button class="give-btn social-btn facebook-btn"
+					onclick="
+							// Retrieve and sanitize url to be shared
```

Comment:
URIs need to be escaped/sanitized

Reply:
@bradp This is not user input so I do not need to escape because we are accessing URL from `window.location` and I think the browser will take care of this.